diff --git a/node_modules/react-native-autocomplete-dropdown/src/Dropdown.js b/node_modules/react-native-autocomplete-dropdown/src/Dropdown.js
index 003dc6d..e8954ac 100644
--- a/node_modules/react-native-autocomplete-dropdown/src/Dropdown.js
+++ b/node_modules/react-native-autocomplete-dropdown/src/Dropdown.js
@@ -25,7 +25,7 @@ export const Dropdown = memo(
           ...(position === 'relative'
             ? { marginTop: 5 }
             : {
-                [direction === 'down' ? 'top' : 'bottom']: inputHeight + 5
+                [direction === 'down' ? 'top' : 'bottom']: inputHeight
               }),
           ...props.suggestionsListContainerStyle
         }}>
diff --git a/node_modules/react-native-autocomplete-dropdown/src/index.js b/node_modules/react-native-autocomplete-dropdown/src/index.js
index 573e255..6ae3c32 100644
--- a/node_modules/react-native-autocomplete-dropdown/src/index.js
+++ b/node_modules/react-native-autocomplete-dropdown/src/index.js
@@ -16,6 +16,7 @@ import { moderateScale, ScaledSheet } from 'react-native-size-matters'
 import { NothingFound } from './NothingFound'
 import { RightButton } from './RightButton'
 import { ScrollViewListItem } from './ScrollViewListItem'
+import Search from './search.svg'
 
 export const AutocompleteDropdown = memo(
   forwardRef((props, ref) => {
@@ -272,6 +273,7 @@ export const AutocompleteDropdown = memo(
           ref={containerRef}
           onLayout={_ => { }}
           style={[styles.inputContainerStyle, props.inputContainerStyle]}>
+            <Search width={16} height={16} marginLeft={12}/>
           <InputComponent
             ref={inputRef}
             value={searchText}
@@ -365,7 +367,9 @@ const styles = ScaledSheet.create({
     display: 'flex',
     flexDirection: 'row',
     backgroundColor: '#e5ecf2',
-    borderRadius: 5
+    borderRadius: 5,
+    alignItems: 'center',
+    justifyContent: 'center',
   },
   Input: {
     flexGrow: 1,
diff --git a/node_modules/react-native-autocomplete-dropdown/src/search.svg b/node_modules/react-native-autocomplete-dropdown/src/search.svg
new file mode 100644
index 0000000..638bda8
--- /dev/null
+++ b/node_modules/react-native-autocomplete-dropdown/src/search.svg
@@ -0,0 +1,3 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="#747476" xmlns="http://www.w3.org/2000/svg">
+<path fill-rule="evenodd" clip-rule="evenodd" d="M10.1878 11.623C9.12979 12.3905 7.82856 12.843 6.4215 12.843C2.87499 12.843 0 9.96802 0 6.4215C0 2.87499 2.87499 0 6.4215 0C9.96802 0 12.843 2.87499 12.843 6.4215C12.843 7.8283 12.3907 9.12953 11.6233 10.1875C11.7242 10.2377 11.8185 10.3048 11.9026 10.389L15.6866 14.173C16.1045 14.5908 16.1045 15.2686 15.6866 15.6864C15.2686 16.1045 14.5911 16.1045 14.173 15.6864L10.3892 11.9026C10.3051 11.8185 10.2379 11.7239 10.1878 11.623ZM10.7025 6.4215C10.7025 8.78593 8.78593 10.7025 6.4215 10.7025C4.05708 10.7025 2.1405 8.78593 2.1405 6.4215C2.1405 4.05708 4.05708 2.1405 6.4215 2.1405C8.78593 2.1405 10.7025 4.05708 10.7025 6.4215Z" fill="#747476"/>
+</svg>
